provers/sp1/driver/Cargo.toml

```diff
@@ -49,13 +51,16 @@ enable = [
     "raiko-lib",
     "sp1-sdk",
     "anyhow",
+    "alloy-contract",
     "alloy-primitives",
+    "alloy-provider",
     "alloy-sol-types",
     "once_cell",
     "sha3",
     "dotenv",
     "serde_with",
     "tokio",
-    "tracing"
+    "tracing",
 ]
 neon = ["sp1-sdk?/neon"]
+foundry-verify = []
```

Comment:
do we need this additional feature to guard this code? Doesn't seem like it adds any dependencies or anything, so may as well always compile it in?

Reply:
currently we use feature `enable` which does not include this one, so the foundry test code will not be included in compiling. The previous foundry path is too heavy as it downloads the sp1 sdk (10G+) and cp necessary files from it (actually does not work because some sol files are broken in their sdk), and then spawn a `forge test` process. I think we can remove it later if no one really uses this forge test mode, so I use new feature flag to keep it for a while.